gtsfm/loader/astronet_loader.py

```diff
@@ -78,9 +75,12 @@ def __init__(
         if not Path(data_dir).exists():
             raise FileNotFoundError("No data found at %s." % data_dir)
         cameras, images, points3d = colmap_io.read_model(path=data_dir, ext=".bin")
-        self._calibrations, self._wTi_list, img_fnames, self._sfmtracks = self.colmap2gtsfm(
-            cameras, images, points3d, load_sfmtracks=use_gt_sfmtracks
-        )
+        (
+            self._calibrations,
+            self._wTi_list,
+            img_fnames,
+            self._sfmtracks,
+        ) = io_utils.colmap2gtsfm(cameras, images, points3d, load_sfmtracks=use_gt_sfmtracks)
```

Comment:
probably needs to be reformatted with the 120 character limit? When you do that, remove the parenthesis on the return values before reformatting.

Reply:
Reverted.